otx/algorithms/common/adapters/torch/dataloaders/samplers/balanced_sampler.py

```diff
@@ -7,9 +8,12 @@
 
 logger = get_logger()
 
+# pylint: disable=too-many-instance-attributes
```

Comment:
To disable pylint as minimum as possible, how about setting in class?

Reply:
We had a consensus that we can ignore too-many-* as of now.